backend/main.go

```diff
@@ -125,20 +161,20 @@ func registerHandler(w http.ResponseWriter, r *http.Request) {
 	}
 	if ssn != respData.SSN || dob != respData.DOB {
 		pay.Message = "Not a citizen"
-		sendResult(w, pay)
+		sendResult(w, pay, true)
```

Comment:
If they're not a citizen, why are we returning a success? Is this success just to say "We talked to the backend successfully"?